tests/test_utils/__init__.py

```diff
@@ -97,27 +104,44 @@ async def cb(*args, **kwargs):
     return Mock(side_effect=cb)
 
 
-@attr.s
-class FakeResponse:
+# Type ignore: it does not fully implement IResponse, but is good enough for tests
+@zope.interface.implementer(IResponse)
```

Comment:
Not sure if that was the right way to do it, but `mypy` seems happy with that

Reply:
This is the correct way to do this! 👍